includes/options/class-amp-options-menu.php

```diff
@@ -71,6 +97,142 @@ public function add_menu_items() {
+					<label for="theme_support_paired">
+						<strong><?php esc_html_e( 'Paired', 'amp' ); ?></strong>
+					</label>
+				</dt>
+				<dd>
+					<?php esc_html_e( 'Reuse active theme\'s templates to display AMP responses, but use separate URLs for AMP. The canonical URLs for your site will not have AMP. If there are AMP validation errors encountered in the AMP response and the validation errors are not accepted for sanitization, then the AMP version will redirect to the non-AMP version.', 'amp' ); ?>
+				</dd>
+				<dt>
+					<input type="radio" id="theme_support_native" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[theme_support]' ); ?>" value="native" <?php checked( $theme_support, 'native' ); ?> <?php disabled( ! $theme_support_mutable ); ?>>
+					<label for="theme_support_native">
+						<strong><?php esc_html_e( 'Native', 'amp' ); ?></strong>
+					</label>
+				</dt>
+				<dd>
+					<?php esc_html_e( 'Reuse active theme\'s templates to display AMP responses but do not use separate URLs for AMP. Your canonical URLs are AMP. Select this if you want to use AMP-specific blocks in your content. Any AMP validation errors will be automatically sanitized.', 'amp' ); ?>
+				</dd>
+			</dl>
+		</fieldset>
+		<?php
+	}
+
+	/**
+	 * Post types support section renderer.
+	 *
+	 * @todo If dirty AMP is ever allowed, then automatically forcing sanitization in native should be able to be turned off.
+	 *
+	 * @since 1.0
+	 */
+	public function render_validation_handling() {
+		?>
+		<fieldset>
+			<?php
+			$forced_sanitization = AMP_Validation_Error_Taxonomy::is_validation_error_sanitized( array(
+				'code' => 'non_existent',
+			) );
+			$forced_tree_shaking = $forced_sanitization || AMP_Validation_Error_Taxonomy::is_validation_error_sanitized( array(
+				'code' => AMP_Style_Sanitizer::TREE_SHAKING_ERROR_CODE,
+			) );
+			?>
+
+			<?php if ( $forced_sanitization ) : ?>
+				<div class="notice notice-info notice-alt inline">
+					<p>
+						<?php esc_html_e( 'Your install is configured via a theme or plugin to automatically sanitize any AMP validation error that is encountered.', 'amp' ); ?>
+					</p>
+				</div>
+				<input type="hidden" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[force_sanitization]' ); ?>" value="<?php echo AMP_Options_Manager::get_option( 'force_sanitization' ) ? 'on' : ''; ?>">
+			<?php else : ?>
+				<div class="amp-force-sanitize">
+					<p>
+						<label for="force_sanitization">
+							<input id="force_sanitization" type="checkbox" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[force_sanitization]' ); ?>" <?php checked( AMP_Options_Manager::get_option( 'force_sanitization' ) ); ?>>
+							<?php esc_html_e( 'Automatically accept sanitization for any AMP validation error that is encountered.', 'amp' ); ?>
```

Comment:
It's great how this conditionally displays on selecting "Paired":
<img width="1440" alt="conditional-display" src="https://user-images.githubusercontent.com/4063887/41018010-5890e80e-691d-11e8-8c52-c8fed3e4a95e.png">